specs/src/shrex/shrex-sub.md

```diff
@@ -118,7 +118,6 @@ Validate(peerID PeerID, message []byte) -> ValidationResult
 
 - `ACCEPT`: Message is valid and MUST be processed
 - `REJECT`: Message is invalid and MUST be discarded
-- `IGNORE`: Message is valid but duplicate/stale and SHOULD be ignored
```

Comment:
why was this removed? it's still a validation result that can/should be returned